.github/workflows/deploy-infra.yaml

```diff
@@ -329,33 +326,30 @@ jobs:
               --install --wait
         env:
           PUBLIC_FACING: ${{ steps.platform-public-facing.outputs.result }}
-          ACM_CERTIFICATE_ARN: ${{ steps.setup-domain.outputs.acm-certificate-arn }}
+          ACM_CERTIFICATE_ARN: ${{ secrets.ACM_CERTIFICATE_ARN }} 
```

Comment:
setting this directly from the secret doesn't work for HMS staging as there are separate certificates for staging (`secrets.ACM_CERTIFICATE_ARN_STAGING`) and production (`secrets.ACM_CERTIFICATE_ARN`)

Reply:
Added a condition to set to ACM_CERTIFICATE_ARN_STAGING otherwise use ACM_CERTIFICATE_ARN